build.sbt

```diff
@@ -133,3 +133,20 @@ lazy val treeLib =
       name := "twxplore-tree-lib",
       skip in publish := true
     )
+
+lazy val treeApp = (project in file("app/tree"))
+  .dependsOn(treeLib % "compile->compile;test->test")
+  .enablePlugins(PlayScala)
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.scalatestplus.play" %% "scalatestplus-play" % "4.0.3" % Test
```

Comment:
Don't need this any more, it's in test-lib. Do the dependsOn test-lib "test->compile".